src/adapters/MeetingsJSONAdapter.js

```diff
@@ -157,6 +164,17 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
       /* eslint-enable no-confusing-arrow */
     );
 
+    const shareEvents$ = fromEvent(document, SHARE_CONTROL).pipe(
```

Comment:
I see that you're adding the `remoteShare` whenever the share control is activated, however, I think this should follow the pattern of the other streams, where you add local/remote screen share stream based on the JSON data.